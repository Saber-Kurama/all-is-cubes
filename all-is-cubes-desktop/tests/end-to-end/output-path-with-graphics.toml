# Tests what happens if `-o` *and* a graphics option are set.
# TODO: We want this to succeed, but it currently does not, in the form of
# ignoring the output file option.
bin.name = "all-is-cubes"
args = "--no-config-files --simplify-log-format --template lighting-bench --seed 0 --graphics print -o test.png --display-size 40x40"
status.code = 0
