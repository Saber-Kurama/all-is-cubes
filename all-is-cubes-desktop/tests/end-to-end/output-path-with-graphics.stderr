[DEBUG] (1) Initialized session [..]
[DEBUG] (1) Initialized game state with [..]
[DEBUG] (1) Initialized desktop-session [..]
